services/web/server/tests/unit/with_dbs/01/test_director_v2.py

```diff
@@ -117,5 +137,72 @@ async def test_get_computation_task(
     assert task_out.state == RunningState.NOT_STARTED
 
 
-async def test_delete_pipeline(client, user_id: PositiveInt, project_id: UUID):
+async def test_delete_pipeline(
+    mocked_director_v2, client, user_id: UserID, project_id: ProjectID
+):
     await director_v2_api.delete_pipeline(client.app, user_id, project_id)
+
+
+@settings(suppress_health_check=[HealthCheck.function_scoped_fixture])
+@given(cluster_create=st.builds(ClusterCreate))
```

Comment:
OMG is this really creating a random ``ClusterCreate`` model ?? 

I guess this is in the line of https://github.com/Zac-HD/hypothesis-jsonschema but with pydantic models, right?
AMAZING!

We should use this systematically everywhere!